CsvParser/csvParser_test.go

```diff
@@ -11,14 +11,14 @@ import (
 
 func Test_CsvParserEntries(t *testing.T) {
 
-	testIncident := new(incident.DefaultIncident)
-	testIncident.Init(time.Date(2017, time.January, 1, 0, 0, 0, 0, time.UTC).AddDate(-1, 0, 0),
-		time.Date(2017, time.January, 1, 0, 0, 0, 0, time.UTC),
-		time.Date(2017, time.January, 1, 0, 0, 0, 0, time.UTC),
-		time.Date(2019, time.April, 1, 0, 0, 0, 0, time.UTC),
-		31.046068,
-		21.27035,
-		0.326146, 4788063)
+	// Create test incident to check result against
+	var testIncident incident.DefaultIncident = incident.DefaultIncident{}
+
+	var gs = time.Date(2017, time.January, 1, 0, 0, 0, 0, time.UTC).AddDate(-1, 0, 0)
```

Comment:
I think for clarity, it would be nice to remove the AddDate() call and just set the date to the actual value you want it to be

Reply:
Removed calls to AddDate in tests